CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Add a CHANGELOG.md file.
+
+## [0.2.0] - 2023-12-28
+
+### Added
+
+- Add MIO0 compression and decompression.
```

Comment:
I don't think we need "Add" in the "Added" sections..unless this is part of the spec or something. It seems fine to just say "MIO0 ..."